### PR TITLE
Make all columns be filterable or groupable

### PR DESCRIPTION
**Pull request by agrawaldevesh**
I want to make every column be filterable and groupable and not just strings.

### CODE REVIEW

File: superset/config.py [hunk 1]
```diff
@@ -188,7 +188,9 @@
 # ---------------------------------------------------
 # Feature flags that are on by default go here. Their
 # values can be overridden by those in super_config.py
-FEATURE_FLAGS = {}
+FEATURE_FLAGS = {
+    'ALL_COLS_FILTERABLE_GROUPABLE': False,
```
Comment (agrawaldevesh) [inline superset/config.py]: @xtinec , Can you review this use of feature flags ?

**Comment by mistercrunch** [comment]
Personally would push for this being the new default, no need for a feature flag

**Comment by agrawaldevesh** [comment]
@mistercrunch, I tried setting the feature flag as default true and it fails the mysql or pg unit tests in https://travis-ci.org/apache/incubator-superset/jobs/485991707 (in tests.sqllab_tests.SqlLabTests#test_search_query_on_time). 

I frankly cannot comprehend these tests: They seem to be doing some meta level searching of the query log themselves. I am not sure why other tests in this file are succeeding with the NO_TABLE table but some fail. Perhaps its to do with some inadvertent state changes, of tests interferring with each other or not cleaning up properly.

Anyway, I think we can keep this flag be FALSE for now and set it to true/default after a while.

File: superset/config.py (hunk 1 above)
Comment (mistercrunch) [reply superset/config.py]: I'm not sure how these feature flags are supposed to work in practice in `superset_config`. It appears there's no good way to alter only one of them. You'd have to either alter all of them or none of them. If you were to copy/paste the whole dict to your `superset_config` you'd miss out on new ones as they come, and if you try to import the dict I think you'd run in circular import issue. @xtinec what do you think?

Comment (mistercrunch) [reply superset/config.py]: @agrawaldevesh let's figure out what's up with postgres, let's not use the feature flag for this specifically or at least until we confirm it's usable. If you repush without the flag I can try and help debug the postgres-specific issue

**Comment by agrawaldevesh** [comment]
@mistercrunch, sure ... it would be best to fix the root test issue :-). I think its a bit flaky. I have seen this fail with mysql once, once with postgres and once all tests passed. And now I am running the tests again: https://travis-ci.org/agrawaldevesh/incubator-superset/builds/486245360

**Comment by mistercrunch** [comment]
Sounds good. Let's get to the bottom of it.

**Comment by mistercrunch** [comment]
LGTM

**Pull request merged**